services/collector/collector.yaml

```
@@ -0,0 +1,712 @@
+exporters:
+  #otlp:
+    #endpoint: collector.opensafely.org:443
```

```suggestion
    #endpoint: https://collector.opensafely.org:443
```
I think if you don't specify http(s), it's going to try to use grpc by default. Anyway we'll find out pretty quickly when we uncomment this.